src/app/modules/group/services/group-datasource.service.ts

```diff
@@ -22,15 +27,19 @@ export class GroupDataSource implements OnDestroy {
   state$ = this.fetchOperation.pipe(
     // switchMap does cancel the previous ongoing processing if a new one comes
     // on new fetch operation to be done: set "fetching" state and fetch the data which will result in a ready or error state
-    switchMap(route => this.getGroupByIdService.get(route.id).pipe(
-      map(group => ({ route, group })),
+    switchMap(route => forkJoin({
+      group: this.getGroupByIdService.get(route.id),
+      breadcrumbs: this.getGroupBreadcrumbsService.getBreadcrumbs(route),
+    }).pipe(
+      map(({ group, breadcrumbs }) => ({ route, group, breadcrumbs })),
```

Comment:
I'll have a serious conflict with master on merge, so you should rebase/merge first.